src/pharmpy/tools/iivsearch/tool.py

```diff
@@ -260,10 +261,16 @@ def start(
         final_model = res.final_model
         if final_model.name != input_model_entry.model.name:
             # FIXME: This should be piped instead
-            final_model_entry = context.retrieve_model_entry(final_model.name)
+            res_path = (
+                context.path
+                / "models"
+                / final_model.name
+                / ("model" + context.model_database.file_extension)
+            )
+            final_res = get_modelfit_results(final_model, res_path)
```

Comment:
Couldn't the code be added to the context so that iivsearch doesn't need to know about this workaround? Perhaps it could even be piped instead as the FIXME suggests.